packages/components/src/components/Address/Address.tsx

```diff
@@ -6,11 +6,27 @@ export interface AddressProps {
    * The address to display
    */
   value: string;
+  /**
+   * boolean to shorten the address if addresses does not resolve to an ENS name
+   */
+  shortened: boolean;
 }
 
 /**
  * A component to display an address
  */
-export const Address: React.FC<AddressProps> = ({ value }) => {
-  return <Input value={value} />;
+export const Address: React.FC<AddressProps> = ({ value, shortened }) => {
```

Comment:
add `import React, { FC } from 'react';` to drop the React here

Reply:
Could we set the `shortened` default value to `false` for readability? it will also display default to false in Storybook

```jsx
({ value, shortened = false })
```

Reply:
made both changes. Thanks!